README.md

```diff
@@ -32,7 +30,7 @@ This is a simple math magic application built with JavaScript, HTML, and CSS. It
 
```

Comment:
| Your MIT Section | Previous Reviewer MIT Section |
|-|-|
| ![image](https://github.com/misikir21/mama/assets/93039359/26a863ae-517c-495a-b42d-5c8eb5690677) | ![image](https://github.com/misikir21/mama/assets/93039359/2de792e2-8ff1-4763-9da4-100e1e2035fa) |

- [ ] It would be nice if you make the MIT section as suggested by the previous reviewer as you can see in the second image above to make your readme descriptive and professional 👍 kindly remove the content of your license from your readme file and make your License section as suggested by the previous reviewer 😊